### DISCUSSION TITLE
Production installation Stopped Ubuntu 18 Network manager Error

### DISCUSSION THREAD

**Discussion by adventurousmarc** [Security-Onion-Solutions/securityonion#1645] [Unsupported Versions] [answered]
When I try to start the GitHub installation of SecurityOnion 2 StandAlone, but I received an error for my monitoring port "enp0s25 is unmanaged by Network Manager. Please remove it from other network management tools then re-run setup" I've tried to remove it numerous times with no luck. maybe I am inputting the wrong command but I am lost. please help

**Comment by weslambert** [comment, marked as answer]
Please try the latest [Security Onion 2.3.10](https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md) and let us know if you still experience issues.
